### PR TITLE
Re-Assignement1

### PR DESCRIPTION
**Pull request by mlaratrullenque**

### CODE REVIEW

**Comment by ocorcho** [comment]
1 is wrong, since you are including wrong data due to the merge of somefiles.
2 is ok
I leave @dgarijo to check assignment4

File: Assignment4/mlaratrullenque-22A249/task06.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1qPtO4AqiGGt6iCsZfsL1GYcI5HD9z-Ps
+
+Task 06: Modifying RDF(s)
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/mlaratrullenque-22A249/task06.py:12]: won't work in python. This is Notebook magic

**Review by dgarijo — APPROVED** [review]
There are a couple of small issues, but approved from my end

**Comment by dgarijo** [comment]
PR cannot be merged due to conflicts in the DatasetDescriptions.csv file

**Comment by ocorcho** [comment]
Doing the conflict resolution myself, to allow for the PR to be included.

**Pull request merged**